molo/core/management/commands/add_language_to_pages.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.core.management.base import BaseCommand
+from molo.core.models import LanguageRelation
+from molo.core.models import Page
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        for relation in LanguageRelation.objects.all():
+            if relation.page and relation.language:
+                page = Page.objects.get(pk=relation.page.pk).specific
+                page.language = relation.language
+                page.save()
+            else:
+                self.stdout.write(self.style.NOTICE(
```

Comment:
Please give a more generic error message so that we know if it's the page or the language that is the issue